### PR TITLE
Fixes required for CloudFormation deploy

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

**Review by axthosarouris — APPROVED** [review]

File: template.yaml [hunk 1]
```diff
@@ -1066,6 +1064,61 @@ Resources:
...
+              - Effect: Allow
+                Action:
+                  - events:PutEvents
+                Resource: !Ref EventBusArn
+              - Effect: Allow
+                Action:
+                  - sqs:SendMessage
+                  - sqs:ReceiveMessage
+                  - sqs:DeleteMessage
+                Resource: !GetAtt DeleteDraftPublicationHandlerDLQ.Arn
```
Comment (axthosarouris) [inline template.yaml:1104]: Are the event-related rights necessary?

File: template.yaml [hunk 2]
```diff
@@ -1066,6 +1064,61 @@ Resources:
...
+        - PolicyName: LambdaDefaults
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                  - lambda:InvokeFunction
```
Comment (axthosarouris) [inline template.yaml:1089]: Maybe the lambda invoke function is not necessary

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 2 above)
Comment (janbj) [reply template.yaml:1089]: The original SAM policy template added to the the default lambda InvokeFunction role so I think this is necessary.

File: template.yaml (hunk 1 above)
Comment (janbj) [reply template.yaml:1104]: If you take a look at the Lambda config you will see that it is necessary:

`      EventInvokeConfig:
        DestinationConfig:
          OnSuccess:
            Type: EventBridge
            Destination: !Ref EventBusArn
          OnFailure:
            Type: SQS
            Destination: !GetAtt DeleteDraftPublicationHandlerDLQ.Arn`

**Review by brinxmat — APPROVED** [review]

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**